<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Status</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="X">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Y">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Diameter</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="Diameter">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="minimum">
              <double>50.000000000000000</double>
             </property>
             <property name="maximum">
              <double>200.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Frequency</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="Frequency">
             <property name="suffix">
              <string> Hz</string>
             </property>
             <property name="minimum">
              <double>0.100000000000000</double>
             </property>
             <property name="maximum">
              <double>2.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="UseExternalData">
           <property name="text">
            <string>Use External Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ToggleLed">
           <property name="text">
            <string>Toggle Led</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Calibration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="goto_0_0">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Go to [0,0]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="goto_100_0">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Go to [100,0]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="goto_0_100">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Go to [0,100]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="goto_100_100">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Go to [100,100]</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart_Motion"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop_Motion"/>
   <addaction name="separator"/>
   <addaction name="actionRead_External_File"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionStart_Motion">
   <property name="text">
    <string>Start Motion</string>
   </property>
  </action>
  <action name="actionStop_Motion">
   <property name="text">
    <string>Stop Motion</string>
   </property>
  </action>
  <action name="actionRead_External_File">
   <property name="text">
    <string>Read External File</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>goto_0_0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Goto_0_0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goto_100_0</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Goto_100_0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goto_0_100</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Goto_0_100()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goto_100_100</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Goto_100_100()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStart_Motion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Start_Motion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop_Motion</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Stop_Motion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRead_External_File</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Read_External_File()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPause</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>On_Pause_Motion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>On_Goto_0_0()</slot>
  <slot>On_Goto_100_0()</slot>
  <slot>On_Goto_0_100()</slot>
  <slot>On_Goto_100_100()</slot>
  <slot>On_Start_Motion()</slot>
  <slot>On_Stop_Motion()</slot>
  <slot>On_Read_External_File()</slot>
  <slot>On_Quit()</slot>
  <slot>On_Pause_Motion()</slot>
 </slots>
</ui>
